docs/Intro/network.md

```diff
@@ -0,0 +1,31 @@
+
+# Decentralized Oracle Network
+You can think of Chronicle as a decentralized protocol managing a network of trusted participants (validators) that report pricing information from various sources (both on-chain and off-chain). One aspect that sets Chronicle apart from other Oracle providers is that the validators of Chronicle are represented by a community of the leading blockchain protocols including [MakerDAO](https://makerdao.com), [Infura](https://www.infura.io/), [Gitcoin](https://www.gitcoin.co/), [dYdX](https://dydx.exchange/), [0x](https://0x.org/), [Etherscan](https://etherscan.io/), [Gnosis](https://www.gnosis.io/), [DeFi Saver](https://defisaver.com/), [My Crypto](https://mycrypto.com/). This approach is unique to Chronicle and ensures that the Oracle data is being signed by some of the most trusted and reputable brands in the blockchain space, further reinforcing trust in the protocol.
+
+Smart contracts often perform real-time operations based on asset price data, particularly in DeFi. For example, MakerDAO uses Chronicle’s Oracles to calculate the value of borrowers' deposits.
+
+Each Oracle has its own peer-to-peer (P2P) network of these participants (validators), who report prices based on strategies (such as source and update threshold) defined by the data model through an on-chain registry.  All validators query external sources, then sign and broadcast (send) those prices assembled into a sorted grouping (the median) by a relay that posts the value on-chain. 
```

Comment:
I'm not sure whether we want to make the config registry public knowledge? Can @NiklasKunkel confirm whether onchain config should be part of docs?

Reply:
ok, seems like config registry is already part of the docs. Resolving....